ems-business-logic/src/main/java/com/devtechgroup/ems/business/logic/service/IEmployeeService.java

```diff
@@ -0,0 +1,15 @@
+package com.devtechgroup.ems.business.logic.service;
+
+import com.devtechgroup.ems.business.logic.model.EmployeeDto;
+
+import java.util.List;
+
+public interface IEmployeeService {
+
+    EmployeeDto createEmployee(EmployeeDto employeeDto);
+    EmployeeDto editEmployee(Long employee_id, EmployeeDto employee);
```

Comment:
Use only camel case for variable. Please avoid naming your variables, methods with underscore (employe_id). Keep in mind it is OK for database column name to be with underscore.